soundata/datasets/tau2021sse_nigens.py

```diff
@@ -359,14 +359,14 @@ def validate_locations(locations):
 
 
 class Clip(core.Clip):
-    """TAU NIGENS SSE 2021 Track class
+    """TAU NIGENS SSE 2021 clip class
```

Comment:
maybe writing `Clip` here (and in the rest of the updated loaders? I mean, using capital C?

Reply:
Sure, will make the changes.